.github/actions/setup-test-environment/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Set up test environment'
+description: 'Set up test environment for mapknitter'
+
+services:
+  mysql:
+    image: mysql:5.7
+    env:
+      MYSQL_ROOT_PASSWORD: root
+    ports:
+    - 3306
+    options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+runs:
+  using: "composite"
+  steps:
+
+  - name: Setup database
+    shell: bash
+    env:
+      RAILS_ENV: test
+      DB_USER: root
+      DB_PASS: root
+      # tell Rails to use proper port for MySQL
+      DB_PORT: ${{ job.services.mysql.ports[3306] }}
+      DISABLE_BOOTSNAP: true
+    run: |
+      cp config/database.yml.example config/database.yml
+      cp db/schema.rb.example db/schema.rb
+      cp config/config.yml.example config/config.yml
+      sudo systemctl start mysql
+      mysql -uroot -proot -e "CREATE DATABASE mapknitter_test;"
+      mysql -uroot -proot -e "SET @@global.sql_mode=(SELECT REPLACE(@@global.sql_mode, 'ONLY_FULL_GROUP_BY', ''));"
```

Comment:
i think this is not necessary... we should try removing later. It was for specific issues with the db version on plots2 only.

Reply:
yeah, that happens when copying code without knowing exactly what it does 😳  🙈 
Thanks!